<ui version="4.0" >
 <class>Config</class>
 <widget class="QDialog" name="Config" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8" >
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok" >
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>XML File</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <item row="0" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3" >
              <item>
               <widget class="QRadioButton" name="fromFile" >
                <property name="text" >
                 <string>From File</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="fromUrl" >
                <property name="text" >
                 <string>From URL</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4" >
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4" >
                <item>
                 <widget class="QLineEdit" name="XmlFilename" />
                </item>
                <item>
                 <widget class="QToolButton" name="XmlFilenameButton" >
                  <property name="text" >
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QComboBox" name="comboURL" >
                <item>
                 <property name="text" >
                  <string>http://xmltv.myftp.org/download/tnt.zip</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>http://xmltv.myftp.org/download/complet.zip</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="populateDB" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Read</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>French XML Files</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" >
          <item row="0" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QLabel" name="label_9" >
              <property name="text" >
               <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Available on &lt;a href="http://xmltv.myftp.org">&lt;span style=" text-decoration: underline; color:#0000ee;">http://xmltv.myftp.org&lt;/span>&lt;/a>, the &lt;a href="http://xmltv.myftp.org/download/tnt.zip">&lt;span style=" text-decoration: underline; color:#0000ee;">TNT&lt;/span>&lt;/a> guide or &lt;a href="http://xmltv.myftp.org/download/complet.zip">&lt;span style=" text-decoration: underline; color:#0000ee;">complets&lt;/span>&lt;/a> for 9 days. &lt;/p>&lt;/body>&lt;/html></string>
              </property>
              <property name="openExternalLinks" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" name="horizontalLayout_6" >
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <item>
            <widget class="QCheckBox" name="systrayStarts" >
             <property name="text" >
              <string>Start on systray</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" >
             <item>
              <widget class="QLabel" name="label" >
               <property name="text" >
                <string>Start Hour:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="startHour" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection" >
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="maximum" >
                <number>23</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" >
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5" >
             <item>
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>Programs height</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3" >
               <property name="text" >
                <string>Half-hour width</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" >
             <item>
              <widget class="QSpinBox" name="programHeight" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="minimum" >
                <number>30</number>
               </property>
               <property name="maximum" >
                <number>99999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="programWidth" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="minimum" >
                <number>50</number>
               </property>
               <property name="maximum" >
                <number>999999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="0" >
        <layout class="QHBoxLayout" name="horizontalLayout_7" >
         <item>
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Search automatically update for</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="findUpdate" >
           <property name="text" >
            <string>QMagneto</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="0" >
        <layout class="QHBoxLayout" name="horizontalLayout_8" >
         <item>
          <widget class="QLabel" name="label_15" >
           <property name="text" >
            <string>Programs Font</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="comboFont" />
         </item>
         <item>
          <widget class="QSpinBox" name="fontSize" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="6" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" >
        <layout class="QHBoxLayout" name="horizontalLayout_9" >
         <item>
          <widget class="QCheckBox" name="proxyEnabled" >
           <property name="text" >
            <string>Proxy HTTP:</string>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="proxyAddress" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_25" >
           <property name="text" >
            <string>Port :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="proxyPort" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum" >
            <number>0</number>
           </property>
           <property name="maximum" >
            <number>65535</number>
           </property>
           <property name="value" >
            <number>3128</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>External tools</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Recording</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" >
          <item row="0" column="0" >
           <layout class="QVBoxLayout" name="verticalLayout_6" >
            <item>
             <layout class="QGridLayout" name="gridLayout" >
              <item row="0" column="0" >
               <widget class="QLabel" name="label_7" >
                <property name="text" >
                 <string>Program:</string>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QLineEdit" name="command" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="label_10" >
                <property name="text" >
                 <string>Options:</string>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QLineEdit" name="commandOptions" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_11" >
              <property name="text" >
               <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">In options, &lt;span style=" font-weight:600;">$STREAM&lt;/span> is the stream and &lt;span style=" font-weight:600;">$OUT&lt;/span> the file created.&lt;/p>&lt;/body>&lt;/html></string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_6" >
              <item row="0" column="0" >
               <widget class="QLabel" name="label_4" >
                <property name="text" >
                 <string>Recording directory:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QLineEdit" name="directory" />
              </item>
              <item row="0" column="2" >
               <widget class="QToolButton" name="directoryButton" >
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="label_5" >
                <property name="text" >
                 <string>Default filename format:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QLineEdit" name="filename" />
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style kind="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%n&lt;/span> : Channel Name &lt;span style=" font-weight:600;">%t&lt;/span> : Program title &lt;span style=" font-weight:600;">%y&lt;/span> : Year &lt;span style=" font-weight:600;">%m&lt;/span> : Month  &lt;span style=" font-weight:600;">%d&lt;/span> : Day&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Reading</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5" >
          <item row="0" column="0" >
           <layout class="QVBoxLayout" name="_9" >
            <item>
             <layout class="QHBoxLayout" name="_10" >
              <item>
               <layout class="QVBoxLayout" name="_11" >
                <item>
                 <widget class="QLabel" name="label_12" >
                  <property name="text" >
                   <string>Program:</string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_13" >
                  <property name="text" >
                   <string>Options:</string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="_12" >
                <item>
                 <widget class="QLineEdit" name="commandLecture" />
                </item>
                <item>
                 <widget class="QLineEdit" name="commandLectureOptions" />
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_14" >
            <property name="text" >
             <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">In options, &lt;span style=" font-weight:600;">$STREAM&lt;/span> is the stream.&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>Config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>Config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XmlFilename</sender>
   <signal>textChanged(QString)</signal>
   <receiver>fromFile</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>350</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel" >
     <x>83</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboURL</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>fromUrl</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>371</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel" >
     <x>83</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>81</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel" >
     <x>252</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>81</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel" >
     <x>444</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
